src/SDKs/_metadata/cognitiveservices_data-plane_CustomImageSearch.txt

```diff
@@ -1,11 +1,11 @@
-2018-05-03 09:18:53 UTC
+2018-09-13 10:23:28 UTC
```

Comment:
Are 2 .txt files being modified when you run the generate.ps1, please undo these changes and regenerate the code, there should be only 1 .txt file modified

Reply:
In that case, assembly versions and PackageReleaseNotes for both SDKs must be updated